src/components/CommentHead/index.js

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+import capitalize from '../../utils/capitalize';
+
+const CommentHead = ({ user, date }) => (
+  <div className="row comment-box__user">
+    <img src={user.image} alt="" className="responsive-img circle" />
```

Comment:
Also if there is no image why don't we show an avatar here too

Reply:
I have added one